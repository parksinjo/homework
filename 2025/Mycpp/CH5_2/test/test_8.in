4
6
USER 11 88 false :DONG-GU, DAEGU:
0
0
