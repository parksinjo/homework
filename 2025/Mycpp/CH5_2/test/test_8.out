
******************************* Main Menu *********************************
* 0.Exit 1.PersonManager(ch3_2, 4)                                        *
* 2.Class:Object(ch3_1) 3.CopyConstructor(ch5_1) 4.AllocatedMember(ch5_2) *
***************************************************************************
Menu item number? 4
Person::Person(...):u 1 70 true :NAMDAEMUN-RO 123, JONGNO-GU, SEOUL, KOREA:

++++++++++++++++ Allocated Member Menu ++++++++++++++++
+ 0.Exit 1.ChangeAddress 2.ChangeMemo 3.UsingMemoMenu +
+ 4.CopyConstructor 5.NullptrMember 6.inputPerson     +
+++++++++++++++++++++++++++++++++++++++++++++++++++++++
Menu item number? 6
u: u 1 70 true :NAMDAEMUN-RO 123, JONGNO-GU, SEOUL, KOREA:
input person information:
old address(NAMDAEMUN-RO 123, JONGNO-GU, SEOUL, KOREA) deleted
USER 11 88 false :DONG-GU, DAEGU:
u: USER 11 88 false :DONG-GU, DAEGU:

++++++++++++++++ Allocated Member Menu ++++++++++++++++
+ 0.Exit 1.ChangeAddress 2.ChangeMemo 3.UsingMemoMenu +
+ 4.CopyConstructor 5.NullptrMember 6.inputPerson     +
+++++++++++++++++++++++++++++++++++++++++++++++++++++++
Menu item number? 0
Person::~Person():USER 11 88 false :DONG-GU, DAEGU:
address, memo_c_str deleted

******************************* Main Menu *********************************
* 0.Exit 1.PersonManager(ch3_2, 4)                                        *
* 2.Class:Object(ch3_1) 3.CopyConstructor(ch5_1) 4.AllocatedMember(ch5_2) *
***************************************************************************
Menu item number? 0
Good bye!!
